charts/irs-environments/int/values.yaml

```
@@ -44,6 +44,30 @@ irs-helm:
     controlplane:
       apikey:
         secret: <path:traceability-irs/data/int/controlplane#apiKey>
+  discovery:
+    endpoint: "" # Currently unknown
```

```suggestion
    endpoint: "https://semantics.int.demo.catena-x.net/discoveryfinder/api/administration/connectors/discovery"
```

And remove the mocks please.